docs/data-management/organizing-metrics/guardrails.md

```diff
@@ -2,18 +2,29 @@
 sidebar_position: 6
 ---
 
-# Guardrail metrics
+# Guardrail cutoffs
 
-A guardrail metric refers to a specific type of metric in Eppo that is set as a standard measurement for every experiment.
-By configuring a metric as a guardrail, it is automatically included in the analysis of all applicable experiments, ensuring consistent tracking and comparison of results.
-This approach enables you to establish a uniform measurement framework, allowing you to monitor and evaluate key performance indicators consistently across various tests, providing valuable insights and facilitating data-driven decision-making processes.
+A guardrail cutoff refers to a downside threshold that can be set on a metric. When the results for the metric breach this threshold, the metric will display a yellow warning state.
```

Comment:
I think the wording here is a bit awkward and confusing. Instead I would reword to something like

The guardrail cutoff metric is yellow until we are fairly certain that the downside risk on this metric is sufficiently small.